sphinx/application/data/concepts/concept-coredb/src/main/sqldelight/chat/sphinx/concept_coredb/SphinxDatabase.sq

```diff
@@ -544,15 +544,14 @@ messageGetAllToShowByChatId:
 SELECT *
 FROM messageDbo
 WHERE chat_id = ?
-AND type IN(0, 2, 3, 5, 6, 14, 15, 25)
+AND type IN(0, 2, 3, 5, 6, 14, 15, 25, 29)
```

Comment:
This returns **all** boost messages, instead of only the ones that will populate a ChatBubble.

Reply:
Query fixed to only return message of type 29 with reply_uuid = NULL